packages/@okta/i18n/src/properties/login.properties

```diff
@@ -760,6 +760,7 @@ oie.okta_verify.authenticator.description = Verify with a code or receive a push
 ## Select authenticator enrollment
 oie.select.authenticators.enroll.title = Set up Authenticators
 oie.select.authenticators.enroll.subtitle = Set up authenticators to ensure that only you have access to your account.
+oie.select.authenticators.enroll.subtitle.custom = Set up authenticators for {0} to ensure that only you have access to your account.
```

Comment:
Looks like `oie.select.authenticators.enroll.subtitle` is a new key in M1 too. How about changing it to `oie.select.authenticators.enroll.subtitle.generic` and the other one to `oie.select.authenticators.enroll.subtitle.specific` ?

Reply:
This would make it inline with other changes.

Reply:
I wasn't a fan of that naming convention TBH. I prefer to use `oie.select.authenticators.enroll.subtitle` and `oie.select.authenticators.enroll.subtitle.custom` as it tells us that its used when we have a custom title. But I am happy to update it to keep it consistent with the other keys.